docs/source/msp.rst

```diff
@@ -309,9 +309,11 @@ Two ways to handle this:
   Configuration of that MSP would consist of a list of root CAs,
   intermediate CAs and admin certificates; and membership identities would
   include the organizational unit (``OU``) a member belongs to. Policies can then
-  be defined to capture members of a specific ``OU``, and these policies may
-  constitute the read/write policies of a channel or endorsement policies of
-  a chaincode. A limitation of this approach is that gossip peers would
+  be defined to capture members of a specific ``OU`` (should be one of: peer, admin, 
```

Comment:
You may additionally specify identities in policies directly by OU (for instance, maybe you want to create an OU for each region, or business unit of your clients). Since these identities are more free-form, creating sane tooling to deal with this is challenging, and doesn't exist yet.

Yes so because it is not possible to create a policy like "OR('Org1MSP.department1')", where department1 is an OU (even after enabling NodeOUs and adding the specific OU to config.yaml); it becomes very necessary to resolve the confusion with respect to fololowing document line:

Policies can then be defined to capture members of a specific OU, and these policies may constitute the read/write policies of a channel or endorsement policies of a chaincode.

The above line should specifically mention that after enabling NodeOUs, it is only the default OUs that can be referenced in access policies. This is what I learnt from my doubt in StackOverflow:

https://stackoverflow.com/questions/60256416/adding-custom-node-ous-with-hyperledger-fabric

As Gari suggests, it is important to be specific in the particular segment of the code.

Reply:
@chintanr97 I think if you change:
 be defined to capture members of a specific ``OU`` (should be one of: peer, admin,..
to
 be defined to capture members of a specific ``role`` (should be one of: peer, admin, 

it would be clearer.

There maybe other places in the documentation that would need to be corrected as well.  I can look into that.

Reply:
@pamandrejko I have pushed another commit to PR making changes as suggested by you. Please let me know if any further changes are needed before merge.

Reply:
@jyellick Can you please check for the same!